docs/source/usage/openshift.rst

```diff
@@ -0,0 +1,76 @@
+OpenShift
+=========
+
+Owners and lessees can easily install OpenShift on hardware managed through ESI through the use of the `OpenShift assisted installer`_. This guide assumes that the relevant roles have been granted access to the required Ironic actions.
+
+Prerequisites
+-------------
+
+* Access to OpenShift assisted installer
+   * You may use https://console.redhat.com or a local installation of the OpenShift assisted installer
+* An HTTP server for the OpenShift ISO generated by the assisted installer
+* Usage of ESI-managed nodes in the 'available' state
+
+The Ironic provisioning network must also be able to reach the HTTP server and the assisted installer. If you are using https://console.redhat.com, the following commmands will provide the appropriate access:
+
+  .. prompt:: bash $
+
+    openstack subnet set --dns-nameserver 8.8.8.8 <provisioning subnet>
+    sudo iptables -t nat -A POSTROUTING -s <provisioning subnet cidr> -j MASQUERADE
```

Comment:
It's important to note that while the `openstack subnet ...` command can be run anywhere with access to the OpenStack API server, the `iptables` command must be run *on the ironic controller* (possibly controllers, plural, in an HA setup).

Reply:
Upon reflection, this is a bit of a hack: someone interacting with ESI as a service isn't necessarily going to have access to the controller and may not be able to make the necessary network changes.

Reply:
Well, both are operator-level actions. I'll clarify in the doc that this is something like an installation step.